src/main/ApplicationUtils.cpp

```diff
@@ -39,6 +39,15 @@ runWithConfig(Config cfg)
         }
     }
 
+    if (cfg.ENHANCED_MANUAL_CLOSE &&
```

Comment:
see my other comment on the new flag: the right checks are already performed if we don't introduce this new config element, so we can remove this

Reply:
I've removed `isStandaloneValidator()` as well.

Reply:
And now I've removed `ENHANCED_MANUAL_CLOSE` altogether.